storage/statedb/cache_redis_test.go

```diff
@@ -146,13 +191,13 @@ func TestNewRedisCache_Timeout(t *testing.T) {
 		ReadTimeout:  redisCacheTimeout,
 		WriteTimeout: redisCacheTimeout,
 		MaxRetries:   0,
-	}), nil}
+	}), nil, nil}
 
 	key, value := randBytes(32), randBytes(500)
 
 	start := time.Now()
 	cache.Set(key, value)
-	assert.Equal(t, redisCacheTimeout, time.Since(start).Round(redisCacheTimeout/2))
+	time.Sleep(100 * time.Millisecond) // redis cache set item asynchronously
```

Comment:
How about setting the sleep time with `redisCacheTimeout`?
```suggestion
	time.Sleep(redisCacheTimeout) // redis cache set item asynchronously
```

Reply:
Thank you for the interesting suggestion, but I am trying to increase `redisCacheTimeout` in https://github.com/klaytn/klaytn/pull/777. The value could be longer than currently suggested if this async set PR is merged. I think it is a little long to use frequently. How do you think?

Reply:
Oh, `cache.Set` cannot be tested in the test since it operates asynchronously. I am gonna remove the 3 lines using `cache.Set`. You let me check this again. Thanks!

Reply:
I cached it to test `redisCache.set` instead of `cache.Set`. Because `redisCache.set` writes synchronously, we can use the previous testing logic.

Reply:
Thank you for kind explanation :)